src/hubmap_translation/neo4j-to-es-attributes.json

```diff
@@ -206,6 +206,9 @@
     },
     "superseded_associated_processed_component_uuids": {
       "es_name": "superseded_associated_processed_component_uuids"
+    },
+    "display_subtype": {
```

Comment:
@kburke thanks for taking this into consideration for this round of updates. Soon we'll refactor out the usage of this mapping json file and use the entity-api schema yaml flag `indexed: false` to exclude the ones that won't be indexed.